src/pages/ThorChainLP/components/ReusableLpStatus/ReusableLpStatus.tsx

```diff
@@ -117,8 +118,12 @@ export const ReusableLpStatus: React.FC<ReusableLpStatusProps> = ({
     const supplyAssets = assets.map(_asset => {
       const amountCryptoPrecision =
         _asset.assetId === thorchainAssetId
-          ? confirmedQuote.runeCryptoDepositAmount
-          : confirmedQuote.assetCryptoDepositAmount
+          ? 'runeCryptoDepositAmount' in confirmedQuote
+            ? confirmedQuote.runeCryptoDepositAmount
+            : confirmedQuote.runeCryptoWithdrawAmount
+          : 'assetCryptoDepositAmount' in confirmedQuote
+          ? confirmedQuote.assetCryptoDepositAmount
+          : confirmedQuote.assetCryptoWithdrawAmount
```

Comment:
The `in` keyword is a bit of a bug generator, can we do explicit checks?

![image](https://github.com/shapeshift/web/assets/125113430/ab7d91fa-864a-436c-b187-b39da89ac816)

Reply:
Addressed across the board in https://github.com/shapeshift/web/pull/6282/commits/426cf5fa7596a1e64d7672b5ef605468a53ece5f.

Reply:
Whoops, didn't commit the helper: https://github.com/shapeshift/web/pull/6282/commits/11e50f3ea3945350952bbf9f88c21f85e7d0d29d